### PR TITLE
Refine command output for FindCommand

### LINKED ISSUE #208: [BUG] `find` command
# Issues

image

- The displayed message seems to concatenate all the messages together which makes it unreadable.
- Also, the first find message does not work since it requires the "n/" prefix.

### PR DESCRIPTION
**Pull request by jereeemyyyy**
# Things Done

* Reorganise the command output message find command.

close #208

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/FindCommand.java [hunk 1]
```diff
@@ -15,14 +19,15 @@ public class FindCommand extends Command {
...
-            + "Example: " + COMMAND_WORD + " alice wong"
-            + "Example: " + COMMAND_WORD + " p/91234567"
-            + "Example: " + COMMAND_WORD + " a/123, Jurong West Ave 6"
-            + "Example: " + COMMAND_WORD + " c/Investment Plan 1";
+            + "Parameters: n/NAME, p/PHONE_NUMBER, a/ADDRESS, c/CLIENT_TYPE \n"
+            + "Examples: \n"
+            + COMMAND_WORD + " " + PREFIX_NAME + " alice wong\n"
+            + COMMAND_WORD + " " + PREFIX_PHONE + " 91234567\n"
+            + COMMAND_WORD + " " + PREFIX_ADDRESS + " 123, Jurong West Ave 6\n"
+            + COMMAND_WORD + " " + PREFIX_CLIENT_TYPE + " Investment Plan 1";
```
Comment (ahmadsyuaib) [inline src/main/java/seedu/address/logic/commands/FindCommand.java:30]: Good changes in making the examples neater!

File: src/main/java/seedu/address/logic/commands/FindCommand.java [hunk 2]
```diff
@@ -15,14 +19,15 @@ public class FindCommand extends Command {
 
     public static final String COMMAND_WORD = "find";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds persons whose name, phone number, "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds clients whose name, phone number, "
```
Comment (ahmadsyuaib) [inline src/main/java/seedu/address/logic/commands/FindCommand.java:22]: Good changes! LGTM!

Comment (L-rrrr) [inline src/main/java/seedu/address/logic/commands/FindCommand.java:22]: Good catch!

**Review by ahmadsyuaib — APPROVED** [review]
Good job Jeremy! Keep it up!

File: src/main/java/seedu/address/logic/commands/FindCommand.java [hunk 3]
```diff
@@ -15,14 +19,15 @@ public class FindCommand extends Command {
...
             + "address or client type contain any of "
             + "all specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
-            + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
-            + "Example: " + COMMAND_WORD + " alice wong"
-            + "Example: " + COMMAND_WORD + " p/91234567"
-            + "Example: " + COMMAND_WORD + " a/123, Jurong West Ave 6"
-            + "Example: " + COMMAND_WORD + " c/Investment Plan 1";
+            + "Parameters: n/NAME, p/PHONE_NUMBER, a/ADDRESS, c/CLIENT_TYPE \n"
```
Comment (L-rrrr) [inline src/main/java/seedu/address/logic/commands/FindCommand.java:25]: Nice change tailored to clienthub

**Comment by L-rrrr** [comment]
Good!

**Pull request merged**